### ISSUE TITLE
in form jscalendar shows a false date

### ISSUE THREAD

**Issue by trichers** [zikula-modules/Clip#170]
If I have a date-field and I want to update a publikation. I click on the jscalendar date-picker and the selected date is far in the future.
I changed the ifFormat in the date.php to %d.%m.%Y, than it works, but I can´t save the publication, because the date-validations gives an alert.

I use Clip 0.9.4

**Comment by trichers** [comment]
I found out, that the problem comes from the german translation. In the german zikula.po file in line 1354:
#: lib/Zikula/Form/Plugin/DateInput.php:104 lib/util/DateUtil.php:855
msgid "%Y-%m-%d"
msgstr "%d.%m.%Y"

It changes the date format in a german format, though Clip can´t read the date correctly anymore.

**Comment by cmfcmf** [comment]
Well, it's still a bug, so I'd like to reopen the issue.

**Comment by trichers** [comment]
Ok. But someone should make an issue at the german translation file of Zikula. I don´t know, where at Github the translations are.

**Comment by cmfcmf** [comment]
No, I think the German translation file is okay. Clip shouldn't use the dateformat filter here.

**Comment by trichers** [comment]
In date.php on line 48:
https://github.com/zikula-modules/Clip/blob/0.9/lib/Clip/Form/Plugin/Date.php
